nfs/mount.sls

```diff
@@ -6,7 +6,7 @@ include:
 {% for m in salt['pillar.get']('nfs:mount', {}).items() %}
 {{ m[1].mountpoint }}:
   mount.mounted:
-    - device: {{ m[1].location }}
+    - device: '{{ m[1].location }}'
```

Comment:
I tend to prefer the `| yaml` or `| json` filter instead of manual quoting to avoid corner case where the data will contains quotes.